x/pki/keeper/msg_server_approve_add_x_509_root_cert.go

```diff
@@ -89,6 +89,9 @@ func (k msgServer) ApproveAddX509RootCert(goCtx context.Context, msg *types.MsgA
 
 		// add to subject -> subject key ID map
 		k.AddApprovedCertificateBySubject(ctx, rootCertificate.Subject, rootCertificate.SubjectKeyId)
+
+		// add to subject key ID -> certificates map
+		k.AddApprovedCertificateBySubjectKeyID(ctx, rootCertificate)
```

Comment:
Please add a test to 'handler_test.go' to ensure that this method correctly modifies this state.